iconCompiler/generateIconsJS/iconsModules.js

```diff
@@ -6,34 +6,72 @@ DON'T EDIT THIS FILE
 It was auto-generated by "iconCompiler" script
 */`
 
+function generatePaths(arr, iconName) {
+  return `[${arr.map((elem) => `{${generateTagObject(elem, iconName)}}`)}]`
```

Comment:
It might be better to split up this one. 
```js
const objectToReturn = arr.map((elem) => {${generateTagObject(elem, iconName)}})
return JSON.Stringify([objectToReturn]) 
```